ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/suppressor/SuppressEverything.java

```diff
@@ -20,11 +20,20 @@
 
 import org.apache.cassandra.cql3.ColumnSpecification;
 
-public class HideAllSuppressor implements ColumnSuppressor
+public class SuppressEverything implements BoundValueSuppressor
 {
     @Override
     public Optional<String> suppress(ColumnSpecification column, ByteBuffer value)
     {
-        return Optional.of("<" + column.type.asCQL3Type() + ">"); // All values should be suppressed
+        return Optional.of(suppressWithType(column)); // All values should be suppressed
+    }
+
+    /**
+     * @param column the column specification
+     * @return the string representation of the suppressed column type
+     */
+    static final String suppressWithType(ColumnSpecification column)
```

Comment:
Nit: would it make sense to have this elsewhere? All other suppressors seems to be dependent on it.
I.e. we could change the interface to be an abstract class instead.

I am fine with keeping it here as well, just a thought though.

Reply:
I'll keep the interface but create an abstract class for these suppress-classes...